### ISSUE TITLE
Implement Filter

### ISSUE THREAD

**Issue by Jordan-Dennis** [ConaireD/TolimanWIP#7]
Hi all,
This is a reminder that we should try and implement a semi-real looking filter for the model. This probably isn't particularly important given the ccurrent binnings but I think it is worth doing anyway. The main challenge is getting a conbination of macro and micro features. This can probably be done by weighting the likelihood of the current value by that of the change in the last if that makes any sense. 
Regards 
Jordan.

**Comment by ConaireD** [comment]
I think that there will be an actual benchtop test to get these values before flight (If you are talking about the bandwidth filter). I think we can just use that - I don't think it changes much /  at all during flight but I could be wrong.

**Comment by Jordan-Dennis** [comment]
Yeah, but since we don't have any data at the moment I wanted to experiment with a model that had a realistic amount of noise for want of a better word. This is low priority though.

**Comment by Jordan-Dennis** [comment]
I didn't mean trying to learn the bandpass of the filter :laughing:, I imagine that would be far too many parameters unless you treated it through finite Fourier Analysis. I mainly want to add it because in the extremes it will (significantly-ish) affect the PSF.

**Comment by ConaireD** [comment]
Ah right - that makes much more sense. Yeah in the extremes it might be an issue - but we should investigate the extent that the filter varies.

**Comment by Jordan-Dennis** [comment]
Yeah exactly. I might even just be worth doing linear edged filters at the high and low end to see what happens. We could use other idealisations as well like Gaussian ect.
